app/src/main/kotlin/com/softteco/template/ui/feature/settings/SettingsScreen.kt

```diff
@@ -100,6 +101,19 @@ private fun ScreenContent(
                 },
             )
             Divider()
+            ListItem(
+                modifier = Modifier.clickable {
+                    context.sendMail(recipient = "softteco.os.dev@gmail.com", subject = "User Inquiry or Feedback")
```

Comment:
Let's move "softteco.os.dev@gmail.com" and "User Inquiry or Feedback" to constants.